src/wallet/KintoWalletFactory.sol

```diff
@@ -96,7 +101,7 @@ contract KintoWalletFactory is Initializable, UUPSUpgradeable, OwnableUpgradeabl
         override
         returns (IKintoWallet ret)
     {
-        require(kintoID.isKYC(owner), "KYC required");
+        require(kintoID.isKYC(owner) && owner == msg.sender, "KYC required");
```

Comment:
so, only the owner of the walletFactory can create accounts?
also, why not using `onlyOwner` modifier?

Reply:
is confusing but owner is the parameter here not the factory owner

Reply:
oh, my bad!